pkg/satellite/auth.go

```diff
@@ -0,0 +1,38 @@
+package satellite
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+
+	"storj.io/storj/pkg/satellite/satelliteauth"
+)
+
+// Signer creates signature for provided data
+type Signer interface {
+	Sign(data []byte) ([]byte, error)
+}
+
+// getHash creates sha256 for provided value
+func getHash(value []byte) ([]byte, error) {
```

Comment:
`hashValue` makes more sense, but use `hash, err := sha256.Sum(value)` directly, no need for the extra func.